<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityDialog</class>
 <widget class="QDialog" name="ActivityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Notification Content:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notificationContent">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="repetitve_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>158</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interval(ms):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="repetitiveRadioButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Repetitive</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="signleShotRadioButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Single-shot</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTimeEdit" name="singleshot_time_edit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>193</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="single_shot_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="single_shot_date">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>223</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <zorder>repetitve_label</zorder>
  <zorder>single_shot_label</zorder>
  <zorder>label</zorder>
  <zorder>notificationContent</zorder>
  <zorder>time</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>singleshot_time_edit</zorder>
  <zorder>single_shot_date</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
